qutip/qip/gates.py

```diff
@@ -488,6 +509,29 @@ def sqrtiswap(N=None, targets=[0, 1]):
                               [0, 0, 0, 1]]), dims=[[2, 2], [2, 2]])
 
 
+def MS_gate(theta, N=None, targets=(0, 1)):
+    """
+    Quantum obejct of a Mølmer–Sørensen gate
+
+    Parameters
+    ----------
+    theta : The duration of the interaction pulse
+    """
+    if targets != (0, 1) and N is None:
+        N = 2
+
+    if N is not None:
+        return expand_oper(MS_gate(theta), N, targets=targets)
+    else:
+        return Qobj(
+            [
+                [np.cos(theta/2.), 0, 0, -1.j*sin(theta/2.)],
+                [0, np.cos(tehta/2.), -1.j*sin(theta/2.), 0],
```

Comment:
`np.cos(tehta/2.)` should give a bug once run: `tehta` --> `theta`

Reply:
Sorry, it seems that I forgot to push the most recent change